README.md

```diff
@@ -1,2 +1,94 @@
+<img align="right" width="350" height="auto" src="https://martinfowler.com/bliki/images/circuitBreaker/state.png">
+
 # fastify-circuit-breaker
-A low overhead circuit breaker for your routes
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)  [![Build Status](https://travis-ci.org/delvedor/easy-breaker.svg?branch=master)](https://travis-ci.org/delvedor/easy-breaker)
+
+A low overhead [circuit breaker](https://martinfowler.com/bliki/CircuitBreaker.html) for your routes.
+
+## Install
+```
+npm i fastify-circuit-breaker
+```
+
+## Usage
+Register the plugin and if needed pass to it some custom option.<br>
+This plugin will ad an `onSend` hook and expose a `circuitBreaker` utility.<br>
+Call `fastify.circuitBreaker()` when declaring the `beforeHandler` option of a route, in this way you will put that very specific route under the *circuit breaking* check.
+```js
+const fastify = require('fastify')()
+
+fastify.register(require('fastify-circuit-breaker'))
+
+fastify.register(function (instance, opts, next) {
+  instance.route({
+    method: 'GET',
+    url: '/',
+    schema: {
+      querystring: {
+        error: { type: 'boolean' },
+        delay: { type: 'number' }
+      }
+    },
+    beforeHandler: instance.circuitBreaker(),
+    handler: function (req, reply) {
+      setTimeout(() => {
+        reply.send(
+          req.query.error ? new Error('kaboom') : { hello: 'world' }
+        )
+      }, req.query.delay || 0)
+    }
+  })
+  next()
+})
+
+fastify.listen(3000, err => {
+  if (err) throw err
+  console.log('Server listening at http://localhost:3000')
+})
+```
+
+### Options
+You can pass the following options to the during the plugin registration, in this way that values will be used in all the routes.
```

Comment:
"You can pass the following options to the during the plugin registration" => "You can pass the following options during the plugin registration"

Reply:
"that values will be used in all the routes" or as default if not overwritten?

Reply:
It is explained few lines below:
*If you pass the options directly to the utility, it will take precedence over the global configuration.*